Zend/zend_observer.c

```diff
@@ -63,16 +63,16 @@ ZEND_API void zend_observer_startup(void)
 	zend_llist_init(&zend_observer_fiber_switch, sizeof(zend_observer_fiber_switch_handler), NULL, 1);
 	zend_llist_init(&zend_observer_fiber_destroy, sizeof(zend_observer_fiber_destroy_handler), NULL, 1);
 
-	zend_observer_fcall_op_array_extension = -1;
+	zend_observer_fcall_function_extension = -1;
```

Comment:
-1 is probably no longer a good check then, I suppose it'll be -1 in general now. Maybe use `0x80000000` instead.